### PR TITLE
add example with page id

### PR DESCRIPTION
**Pull request by franzholz**
Only one member variable of $TSFE is listed.
Where should the other parts of $TSFE be described?

---------

    /**
     * The page id (int)
     * @var string
     */
    public $id = '';

    /**
     * The type (read-only)
     * @var int|string
     */
    public $type = '';

    /**
    /**
     * Page will not be cached. Write only TRUE. Never clear value (some other
     * code might have reasons to set it TRUE).
     * @var bool
     */
    public $no_cache = false;

    /**
     * The rootLine (all the way to tree root, not only the current site!)
     * @var array
     */
    public $rootLine = [];

    /**
     * The pagerecord
     * @var array
     */
    public $page = [];

    /**
     * This will normally point to the same value as id, but can be changed to
     * point to another page from which content will then be displayed instead.
     * @var int
     */
    public $contentPid = 0;

### CODE REVIEW

File: Documentation/Conditions/Index.rst [hunk 1]
```diff
@@ -1141,6 +1141,10 @@ getTSFE
 
       [getTSFE().type == 98]
 
+   Current page id equals to 17. :ref:`PAGE`::
```
Comment (mbrodala) [inline Documentation/Conditions/Index.rst:1144]: Linking to the explanation of the `PAGE` TypoScript object does not add any value here IMO.

**Review by mbrodala — CHANGES_REQUESTED** [review]
Actually `page["uid"]` should be just fine, there is technically no difference between these two. So I think documenting other ways to access the same value will only lead to confusion.

File: Documentation/Conditions/Index.rst (hunk 1 above)
Comment (franzholz) [reply Documentation/Conditions/Index.rst:1144]: https://docs.typo3.org/m/typo3/reference-typoscript/master/en-us/Setup/Page/Index.html

Only here you have an explanation for the id of the page.

Comment (franzholz) [reply Documentation/Conditions/Index.rst:1144]: Before TYPO3 10 you could write the TypoScript Condition like this:

    [globalVar = TSFE:id = 279]

So I find this solution:

    [getTSFE().id == 279]

I could not find your solution at all.

Another hint could be added in order to refer to page based conditions.

Comment (mbrodala) [reply Documentation/Conditions/Index.rst:1144]: Exactly: https://docs.typo3.org/m/typo3/reference-typoscript/10.4/en-us/Conditions/Index.html#page

Comment (franzholz) [reply Documentation/Conditions/Index.rst]: I have added the local link to page instead.

File: Documentation/Conditions/Index.rst [hunk 2]
```diff
@@ -1141,6 +1141,10 @@ getTSFE
 
       [getTSFE().type == 98]
 
+   Current page id equals to 17. However the recommended way is with :ref:`condition-page`::
```
Comment (mbrodala) [inline Documentation/Conditions/Index.rst:1144]:
```suggestion
   Current page id equals to 17. However, :ref:`condition-page` should be preferred::
```

**Review by mbrodala — CHANGES_REQUESTED** [review]

**Review by mbrodala — APPROVED** [review]

**Pull request merged**